src/pyspiffe/bundle/jwt_bundle/jwt_bundle_set.py

```diff
@@ -1,28 +1,43 @@
+"""
+This module manage JwtBundleSet objects.
+"""
+
 from typing import Mapping
 from pyspiffe.bundle.jwt_bundle.jwt_bundle import JwtBundle
 from pyspiffe.spiffe_id.trust_domain import TrustDomain
 
 
 class JwtBundleSet(object):
-    bundles: Mapping[TrustDomain, JwtBundle]
+    """JwtBundleSet is a set of JWTBundles objects, keyed by trust domain."""
 
-    def new_set(
-        self, bundles: Mapping[TrustDomain, JwtBundle]
-    ) -> Mapping[TrustDomain, JwtBundle]:
-        """
-        NewSet creates a new, empty set.
-        """
-        raise Exception('not implemented')
+    def __init__(self, bundles: Mapping[TrustDomain, JwtBundle]) -> None:
+        """Creates a new initialized with the given JWT bundles.
+
+        Args:
+            bundles: a set of JwtBundles to initialize the JwtBundleSet
 
-    def add(self, bundle: JwtBundle):
         """
-        Add adds a new bundle into the set. If a bundle already exists for the
-        trust domain, the existing bundle is replaced.
+        self._bundles = bundles
+        pass
+
+    def add(self, jwt_bundle: JwtBundle):
+        """Adds a new bundle into the set.
+
+        If a bundle already exists for the trust domain, the existing bundle is
+        replaced.
+
+        Args:
+            jwt_bundle: The new JwtBundle to add
         """
         raise Exception('not implemented')
```

Comment:
error messages should also end with a dot. I understand this is not from this PR but.. if you are willing to make the change. :)